Disallow more stuff in robots.txt and use Crawl-delay option

zas opened the pull request.

yvanzo commented:
@brainzbot, add to whitelist

yvanzo commented on root/robots.txt.production:
```diff
@@ -5,12 +5,14 @@ Disallow: /ajax/
 Disallow: /area/
 Disallow: /cdstub/
 Disallow: /cdtoc/attach
+Disallow: /collection/create
```
This will be covered by https://github.com/metabrainz/musicbrainz-server/pull/515/files#diff-d18cb9e7c5f98fd00839f1bfa79034e9R8

mwiencek commented on root/robots.txt.production:
```diff
@@ -19,15 +21,36 @@ Disallow: /register
 Disallow: /reports
 Disallow: /search
+Disallow: /set-beta-preference
+Disallow: /statistics/timeline
```
How come? We don't really link it anywhere except the menu and it seems useful to show up in searches for "musicbrainz timeline" etc.

mwiencek commented:
I think we should block `/set-language/` too.

zas commented:
> How come? We don't really link it anywhere except the menu and it seems useful to show up in searches for "musicbrainz timeline" etc.

Oops, i added this one by error, fixed
